### PR TITLE
Update events json to include end time and take more parameters.

### PR DESCRIPTION
**Pull request by liuche**

### CODE REVIEW

**Comment by liuche** [comment]
[As a user I do not wish to be informed on events in the past.](https://trello.com/c/Wtbbkup6/164-as-a-user-i-do-not-wish-to-be-informed-on-events-in-the-past)

**Comment by liuche** [comment]
I also added another commit on top that schedules regular pulls from our google calendars.

Made a Prox Hawaii calendar with write privs to limited people: calendar id is  mozilla.com_avh8q3pubnr4uj419aaubpat2g@group.calendar.google.com

**Comment by liuche** [comment]
[As a user I want my app to be in sync with the events stored in the firebase](https://trello.com/c/5PYxdd1V/212-as-a-user-i-want-my-app-to-be-in-sync-with-the-events-stored-in-the-firebase)

**Comment by liuche** [comment]
[As a user, I want to have high-quality, up-to-date events for Kona.](https://trello.com/c/9kuZld3M/223-as-a-user-i-want-to-have-high-quality-up-to-date-events-for-kona)

File: app/representation.py [hunk 1]
```diff
@@ -148,12 +148,14 @@ def _yelpHoursRecord(hours):
...
-def eventRecord(yelpId, lat, lon, title, startTime, url):
+def eventRecord(yelpId, lat, lon, title, startTime, endTime, url):
     return {
             "id": yelpId,
             "coordinates": { "lat": lat, "lng": lon },
             "description": title,
+            "notification": "Event nearby: " + title, # Placeholder for event notification
             "startTime": startTime,
+            "endTime": endTime,
             "url": url
```
Comment (jhugman) [inline app/representation.py:159]: I'm a little bit worried about timezones here. We're storing date-times in local time? If so, then we should probably be explicit about this.

File: app.py [hunk 2]
```diff
@@ -20,11 +20,12 @@ def placeUser(latitude, longitude):
     except KeyboardInterrupt:
         log.info("GOODBYE")
         sys.exit(1)
-    except Exception, err:
+    except Exception as err:
         from app.util import log
         log.exception("Unknown exception")
         abort(500)
```
Comment (jhugman) [inline app.py:26]: We're still seeing these inconsistencies. I've started just catching them and logging them like so:

``` python
except Exception:
     log.exception("Uncaught exception")
```

so we don't even need the `err` in scope.

File: app/events.py [hunk 3]
```diff
@@ -1,17 +1,49 @@
...
+import time
 
 import app.search as search
 from app.clients import yelpClient, googleapikey, eventfulkey
+from app.request_handler import writeEventRecord
 import app.representation as representation
 
 CALENDAR_URL = 'https://www.googleapis.com/calendar/v3/calendars/{}/events?key={}'
+CALENDAR_IDS = [ 'hawaii247.com_dm8m04hk9ef3cc81eooerh3uos@group.calendar.google.com', 'mozilla.com_avh8q3pubnr4uj419aaubpat2g@group.calendar.google.com' ]
+
```
Comment (jhugman) [inline app/events.py:14]: Nit: these should probably start going into `app/constants.py`.

**Review by jhugman — APPROVED** [review]
The biggest thing I'm concerned about is that we're storing timezoned timestamps in the database. 

We should be careful to document this, including possibly the exact timeformat that we're storing in the database.

File: app/representation.py (hunk 1 above)
Comment (liuche) [reply app/representation.py:159]: The content that we receive from the services are in local time (Eventful, the google calendar). This is a good point though, and I'll update the schema to be explicit about it.

**Pull request merged**